### PR TITLE
fix(#291): update crab icon on overview section

### PR DESCRIPTION
**Pull request by chunqiuyiyu**

### CODE REVIEW

File: packages/react-darwinia/src/components/TokenIcon/TokenIcon.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ const styles = `
 const tokenMap = {
   ring: ringImg,
   cring: cringImg,
+  crab: cringImg,
```
Comment (hackfisher) [inline packages/react-darwinia/src/components/TokenIcon/TokenIcon.tsx]: `cring` has already be renamed to `crab`, we should not use `cring` in codebase anymore.

Comment (chunqiuyiyu) [reply packages/react-darwinia/src/components/TokenIcon/TokenIcon.tsx]: ok

File: packages/react-darwinia/src/components/TokenIcon/TokenIcon.tsx [hunk 2]
```diff
@@ -30,6 +30,7 @@ const styles = `
 const tokenMap = {
   ring: ringImg,
   cring: cringImg,
```
Comment (WoeOm) [inline packages/react-darwinia/src/components/TokenIcon/TokenIcon.tsx]: 可以使用 `crab` 替换 `cring` 字段， `cring` 已经不再使用。

Comment (chunqiuyiyu) [reply packages/react-darwinia/src/components/TokenIcon/TokenIcon.tsx]: 好

**Review by WoeOm — APPROVED** [review]

**Pull request merged**